### PR TITLE
New README figure

### PR DESCRIPTION
**Pull request by jpdefrutos**
First draft of the README figure

### CODE REVIEW

**Comment by andreped** [comment]
Wow! This is fantastic! Love it!

File: assets/SMLP.svg
Comment (andreped) [inline assets/SMLP.svg:1]: Would be great if you could move the brains a little closer, such that the figure itself is more compact.

**Review by andreped — CHANGES_REQUESTED** [review]

File: assets/SMLP.svg
Comment (jpdefrutos) [reply assets/SMLP.svg:1]: Uploaded new figure in https://github.com/andreped/super-ml-pets/pull/47/commits/09d1a5ddd3765250c624c9d9a87d2921904cbdb6

**Comment by jpdefrutos** [comment]
Uploaded new figure in https://github.com/andreped/super-ml-pets/pull/47/commits/09d1a5ddd3765250c624c9d9a87d2921904cbdb6

**Review by andreped — APPROVED** [review]
LGTM. Great job!

**Pull request merged**